### ISSUE TITLE
 'System.TypeInitializationException' exception when creating a mapper

### ISSUE THREAD

**Issue by fleed** [agileobjects/AgileMapper#61]
When creating a new mapper using `Mapper.CreateNew()` I get the following exception:

```
An exception of type 'System.TypeInitializationException' occurred in AgileObjects.AgileMapper.dll but was not handled in user code: 'The type initializer for 'AgileObjects.AgileMapper.TypeConversion.ToNumericConverter`1' threw an exception.'
 Inner exceptions found, see $exception in variables window for more details.
 Innermost exception 	 System.Reflection.AmbiguousMatchException : Multiple methods found with 2 parameter(s)
   at AgileObjects.NetStandardPolyfills.MethodExtensionsPolyfill.WithParameterCount(IEnumerable`1 methods, Int32 parameterCount)
   at AgileObjects.AgileMapper.TypeConversion.TryParseConverter`1..ctor()
   at AgileObjects.AgileMapper.TypeConversion.ToNumericConverter`1..cctor()
```

I'm using it in a unit test project (**xunit**) in a `netcoreapp2.1` project.

**Comment by SteveWilkes** [comment]
Hullo,

Thanks for the feedback - I'll check this out shortly. It seems likely (at least plausible) to be down to new method overloads added in .NET Core 2.1 - if it's easy for you to check, could you confirm if you get the same error in netcoreapp2.0?

**Comment by SteveWilkes** [comment]
This has been fixed in the code - new TryParse overloads using Span<T>s were indeed added in .NET Core 2.1.

Release to follow.

**Comment by SteveWilkes** [comment]
Fixed in [v0.23.3](https://www.nuget.org/packages/AgileObjects.AgileMapper).

Thanks for the feedback! :)

**Comment by fleed** [comment]
Thank you @SteveWilkes , I can confirm that it works like a charm now.
